specifyweb/frontend/js_src/lib/components/Forms/SpecifyForm.tsx

```diff
@@ -82,6 +82,29 @@ export function SpecifyForm<SCHEMA extends AnySchema>({
     'flexibleColumnWidth'
   );
   const [language] = userPreferences.use('form', 'schema', 'language');
+
+  const [focusFirstFieldPref] = userPreferences.use(
+    'form',
+    'behavior',
+    'focusFirstField'
+  );
+  const formRef = React.useRef<HTMLDivElement | null>(null);
+
+  React.useEffect(() => {
+    if (focusFirstFieldPref) {
```

Comment:
(optional) Just a personal opinion

In cases like this you can make the code more readable by replacing your condition with the following:
```js
if (!focusFirstFieldPref) return undefined;
```

That way you don't need `{` and `}`, and the has one less indentation level